### PR TITLE
CORE-7691: added a script to register existing apps in the permission…

### PR DESCRIPTION
**Pull request by dennisroberts71**
…s database.

This utility can be used in the development and QA environments where we want to register existing apps, but not retain analyses or apps from before.

### CODE REVIEW

**Comment by dennisroberts71** [comment]
@johnworth already reviewed the new utility, but I also made some Ansible changes, so if someone could look over those, it would be helpful.

File: ansible/roles/de-init-app-permissions/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```
Comment (psarando) [inline ansible/roles/de-init-app-permissions/README.md]: Looks like this template file still needs editing...

**Comment by psarando** [comment]
Other than my minor documentation comment, I didn't catch any issues with the new ansible playbooks/roles :shipit:

File: ansible/roles/de-init-app-permissions/README.md (hunk 1 above)
Comment (dennisroberts71) [inline ansible/roles/de-init-app-permissions/README.md]: Thanks. I'll fix that.

**Pull request merged**